CONTRIBUTING.md

```diff
@@ -108,11 +108,30 @@ Additions to the [developer toolkit][developer] change the core performance, and
 
 This project follows the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specification for PR titles. Conventional Commits make it easier to understand the history of a project and facilitate automation around versioning and changelog generation.
 
+## Running with OpenTelemetry
+
+Goose uses HTTP to access model providers. If you are receiving failures, it can be helpful to see traces
+of the underlying HTTP requests. For example, a 404 could be indicative of an incorrect URL or a missing model.
+
+First, ensure you have an OpenTelemetry compatible collector listening on port 4318, such as [otel-tui][otel-tui].
+
+```bash
+brew install ymtdzzz/tap/otel-tui
```

Comment:
From what I can tell there aren't any macos specific instructions in this file so maybe we should make this more general. But I don't know what's best of ideas like

- `docker run` with port exposed
- just link to the repo
- `go run`
- `go install`

I ordered them in priority of what I may go with for a python project

Reply:
great idea, went with docker